fastlane/swift/SocketClient.swift

```diff
@@ -266,11 +294,13 @@ extension SocketClient: StreamDelegate {
         self.send(rubyCommand: shutdownCommand)
     }
     
-    func processResponse(string: String) {
+    func processResponse(string: String, socket: SocketClient) {
         guard string.count > 0 else {
-            self.socketDelegate?.commandExecuted(serverResponse: .malformedResponse)
-            self.handleFailure(message: ["empty response from ruby process"])
-
+            self.socketDelegate?.commandExecuted(serverResponse: .malformedResponse) {
+                self.handleFailure(message: ["empty response from ruby process"])
+                print("Leave \(#function) \(#line)")
```

Comment:
Either delete or 👇 ?

```suggestion
                verbose(message: "Leave \(#function) \(#line)")
```